components/ui/LevelWindow.js

```diff
@@ -35,26 +36,44 @@ const Wrapper = styled.div`
 
 const Title = styled.div`
   display: flex;
+  user-select: none;
   justify-content: space-between;
   align-items: center;
-  font-family: ChicagoFLF;
+  font-family: ChicagoFLFRegular;
   font-style: normal;
   font-weight: normal;
   font-size: 32px;
-  line-height: 41px;
-  padding: 16px;
+  height: 6.6rem;
```

Comment:
Maybe a min-height needed here... When I add scroll bars in Storybook it collapses to the content:

![Screen Shot 2021-12-28 at 11 43 49](https://user-images.githubusercontent.com/38808/147601674-8225c296-5ab8-4baa-9d41-c59786424f3a.png)

Reply:
Great suggestion!

<img width="859" alt="Screen Shot 2021-12-28 at 6 26 35 PM" src="https://user-images.githubusercontent.com/204331/147616698-9c56adbd-2696-462f-bb20-c551a3ececa5.png">